src/lib/misc/roughtime/info.txt

```diff
@@ -0,0 +1,9 @@
+<defines>
+ROUGHTIME -> 20190220
+</defines>
+
+<requires>
+rng
+ed25519
+socket
```

Comment:
Since this uses SHA-512 directly it should give that as a dependency. (`sha2_64`)